pkg/local/cedarcoremock/threat.go

```diff
@@ -0,0 +1,59 @@
+package cedarcoremock
+
+import (
+	"github.com/guregu/null/zero"
+
+	"github.com/cmsgov/easi-app/pkg/models"
+)
+
+var mockThreats = []*models.CedarThreat{
+	{
+		AlternativeID:     zero.StringFrom("altID1"),
+		ControlFamily:     zero.StringFrom("controlFamily1"),
+		DaysOpen:          1,
+		ID:                zero.StringFrom("threat1"),
+		ParentID:          zero.StringFrom("{11AB1A00-1234-5678-ABC1-1A001B00CC0A}"),
+		Type:              zero.StringFrom("called in"),
+		WeaknessRiskLevel: zero.StringFrom("quite weak"),
+	},
+	{
+		AlternativeID:     zero.StringFrom("altID2"),
+		ControlFamily:     zero.StringFrom("controlFamily2"),
+		DaysOpen:          2,
+		ID:                zero.StringFrom("threat2"),
+		ParentID:          zero.StringFrom("{11AB1A00-1234-5678-ABC1-1A001B00CC1B}"),
+		Type:              zero.StringFrom("fake"),
+		WeaknessRiskLevel: zero.StringFrom("featherweight"),
+	},
+	{
+		AlternativeID:     zero.StringFrom("altID3"),
+		ControlFamily:     zero.StringFrom("controlFamily3"),
+		DaysOpen:          3,
+		ID:                zero.StringFrom("threat3"),
+		ParentID:          zero.StringFrom("{11AB1A00-1234-5678-ABC1-1A001B00CC2C}"),
+		Type:              zero.StringFrom("scary"),
+		WeaknessRiskLevel: zero.StringFrom("critical"),
+	},
+	{
+		AlternativeID:     zero.StringFrom("altID4"),
+		ControlFamily:     zero.StringFrom("controlFamily4"),
+		DaysOpen:          4,
+		ID:                zero.StringFrom("threat4"),
+		ParentID:          zero.StringFrom("{11AB1A00-1234-5678-ABC1-1A001B00CC3D}"),
+		Type:              zero.StringFrom("huge"),
+		WeaknessRiskLevel: zero.StringFrom("immediate"),
+	},
+	{
+		AlternativeID:     zero.StringFrom("altID5"),
+		ControlFamily:     zero.StringFrom("controlFamily5"),
+		DaysOpen:          5,
+		ID:                zero.StringFrom("threat5"),
+		ParentID:          zero.StringFrom("{11AB1A00-1234-5678-ABC1-1A001B00CC4E}"),
+		Type:              zero.StringFrom("silly"),
+		WeaknessRiskLevel: zero.StringFrom("neutralized"),
```

Comment:
While I don't think it seems to _break_ anything, I think we should mock these out (and potentially leave a comment) to line up with expectations from the frontend

These blocks of code are how the frontend parses threat.weaknessRiskLevel into specific sections by threat level

https://github.com/CMSgov/easi-app/blob/d74ac6b0d185d2ce6e55f9146f335d4c4a491b07/src/constants/systemProfile.ts#L18-L25

https://github.com/CMSgov/easi-app/blob/d74ac6b0d185d2ce6e55f9146f335d4c4a491b07/src/views/SystemProfile/components/ATO/index.tsx#L56

I'd just recommend making some of these line up with those types. Perhaps this is something we could/should handle on the backend in the future, but for now, just editing this mock data is a good first step!

Reply:
Yep, we could create these consts on the backend for sure in the future!